### PR TITLE
update dependencies

### LINKED ISSUE #9: Symfony4 support
Can you add support for Symfony4? Otherwise we can't use PHP-FFMpeg/PHP-FFMpeg on SF4

### PR DESCRIPTION
**Pull request by jens1o**
fixes #9 and https://github.com/PHP-FFMpeg/PHP-FFMpeg/issues/470

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -9,15 +9,20 @@
         }
     ],
     "require": {
-        "php": "^5.3.9 || ^7.0",
-        "symfony/filesystem": "^2.3 || ^3.0"
+        "php": "^7.1",
```
Comment (romainneutron) [inline composer.json:12]: why?

Comment (jens1o) [reply composer.json:12]: requirement for phpunit

Comment (romainneutron) [reply composer.json:12]: This lib supports PHP 5 :)

Comment (romainneutron) [reply composer.json:12]: Please keep in mind that there is no value to force a PHP requirement bump, especially when there's no need in the lib itself.

If you do want to bump the PHP version, you should do it in a major release.

Comment (jens1o) [reply composer.json:12]: Alright! The only problem is, that PHPUnit no longer supports 5.x

image

That's why I updated the requirement. I'm having no problem to file a major release :)

Comment (jens1o) [reply composer.json:12]: //push

**Comment by alexander-schranz** [comment]
Thinks this can now be closed as the tests were fixed in #15

**Pull request closed without merging**